pkg/cmd/render/render.go

```diff
@@ -73,6 +79,9 @@ func (r *renderOpts) AddFlags(fs *pflag.FlagSet) {
 	fs.StringArrayVar(&r.etcdServerURLs, "manifest-etcd-server-urls", r.etcdServerURLs, "The etcd server URL, comma separated.")
 	fs.StringVar(&r.etcdServingCA, "manifest-etcd-serving-ca", r.etcdServingCA, "The etcd serving CA.")
 	fs.StringVar(&r.clusterConfigFile, "cluster-config-file", r.clusterConfigFile, "Openshift Cluster API Config file.")
+
+	fs.StringVar(&r.certRotationBase, "unsupported-cert-rotation-base", r.certRotationBase, "UNSUPPORTED! Certificate rotation base unit.")
```

Comment:
isn't there a time.Duration variant for that?

Reply:
Good point, it's worth this being `time.Duration`.